project.py

```diff
@@ -46,8 +53,19 @@ def __init__(self, n_slides=0,
         # ratio of similarity between images based on which we decide if we
         # are going to save an image
         self.similarity = similarity
+        self.__func_keyword_to_function__(compare_function)
         self.load_config()
 
+    def __func_keyword_to_function__(self, keyword):
+        switcher = {
+            'dhash': '__compute_dhash__',
+            'phash': '__compute_phash__',
+            'ahash': '__compute_ahash__',
+            'orb': None
```

Comment:
I see it now.

It would be nice to generalize ORB too but it seems unnecessary in this case.